src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -251,7 +251,7 @@ const useCloudEnvironmentPolling = (googleProject, workspace) => {
       // v1 workspaces: cloud environment (runtime + disk) is used across workspaces that share the same googleProject
       // v2 workspaces: 1 cloud environment per workspace - saturnWorkspaceName == workspaceName and saturnWorkspaceNamespace == Terra Billing Project (which is no longer equivalent to googleProject)
       // TODO: after PPW migration - we should only need saturnWorkspaceName and saturnWorkspaceNamespace labels
-      const cloudEnvFilters = _.pickBy(l => !_.isUndefined(l), saturnWorkspaceVersion === 'v1' ? { creator: getUser().email, googleProject } : { creator: getUser().email, saturnWorkspaceName, saturnWorkspaceNamespace })
+      const cloudEnvFilters = _.pickBy(l => !_.isUndefined(l), saturnWorkspaceVersion === 'v1' ? { role: 'creator', googleProject } : { creator: getUser().email, saturnWorkspaceName, saturnWorkspaceNamespace })
```

Comment:
What about the usage at line 296? 
https://github.com/DataBiosphere/terra-ui/blob/1fea8ec729bb13da97dd5321b34292d46c24703c/src/pages/workspaces/workspace/WorkspaceContainer.js#L296